src/main/java/seedu/address/model/person/Course.java

```diff
@@ -32,6 +32,10 @@ public static boolean isValidCourseName(String test) {
         return test.matches(VALIDATION_REGEX);
     }
 
+    public String getCourseName() {
+        return courseName;
+    }
+
```

Comment:
Not currently required since courseName is actually public, but it is a good idea nonetheless, in case we change accessibility settings.